test/partiql-tests-runner/src/test/kotlin/org/partiql/runner/executor/EvalExecutor.kt

```diff
@@ -12,7 +12,6 @@ import org.partiql.eval.PartiQLStatement
 import org.partiql.lang.eval.CompileOptions
```

Comment:
The changes in this file were taken from [partiql-plugin-impl](https://github.com/partiql/partiql-lang-kotlin/tree/partiql-plugin-impl) to fix the build. Seems like the `partiql-eval` branch build was not working since the merge from `main` commit da03142909b808486e0f170adb52f55d0c0c1eb5.

Reply:
partiql-eval is fixed now

Reply:
I shall rebase